### PR TITLE
In 150681224 cross report county

### PR DESCRIPTION
**Pull request by AmanShurafa**
counties with no in-n-out

### Pivotal Story
- [Add County dropdown list to the Cross Report card #150681224](https://www.pivotaltracker.com/story/show/150681224)

### Purpose
Add a select field with a list of counties at the top of Cross Reports card

### Testing Notes
Field shows up only in edit mode.

### CODE REVIEW

File: app/javascript/screenings/CrossReportEditView.jsx [hunk 1]
```diff
@@ -141,6 +144,30 @@ export default class CrossReportEditView extends React.Component {
         <div className='row col-md-12'>
           <label>This report has cross reported to:</label>
         </div>
+        <div className='row'>
+          <SelectField
+            gridClassName='col-md-6'
+            id='cross_report_county'
+            label='County'
+            onBlur={(event) => {
```
Comment (jgedeon-osi) [inline app/javascript/screenings/CrossReportEditView.jsx]: Do we need to do anything for this on blur at this time?

File: config/environments/production.rb [hunk 2]
```diff
@@ -62,7 +62,6 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
-  # TODO: Implement mail
```
Comment (jgedeon-osi) [inline config/environments/production.rb]: how did you find this? 👍 for removing it

File: spec/javascripts/components/screenings/crossReports/CrossReportEditViewSpec.jsx [hunk 3]
```diff
@@ -89,7 +101,7 @@ describe('CrossReportEditView', () => {
     })
   })
 
-  it('renders labels for required field as required', () => {
+  it('renders labels for required fields as required', () => {
```
Comment (jgedeon-osi) [inline spec/javascripts/components/screenings/crossReports/CrossReportEditViewSpec.jsx]: This test is only testing one field, why change the test name? Or should it be testing more than one field?

File: spec/javascripts/components/screenings/crossReports/CrossReportEditViewSpec.jsx [hunk 4]
```diff
@@ -204,6 +225,7 @@ describe('CrossReportEditView', () => {
       props = {
         isAgencyRequired: jasmine.createSpy('isAgencyRequired').and.returnValue(true),
         crossReports: Immutable.fromJS([{
+          county: 'gotham',
```
Comment (jgedeon-osi) [inline spec/javascripts/components/screenings/crossReports/CrossReportEditViewSpec.jsx]: NaNaNaNa Batman!

File: app/javascript/screenings/CrossReportEditView.jsx (hunk 1 above)
Comment (AmanShurafa) [reply app/javascript/screenings/CrossReportEditView.jsx]: Good catch

File: config/environments/production.rb (hunk 2 above)
Comment (AmanShurafa) [reply config/environments/production.rb]: New IDE 😎

File: spec/javascripts/components/screenings/crossReports/CrossReportEditViewSpec.jsx (hunk 3 above)
Comment (AmanShurafa) [reply spec/javascripts/components/screenings/crossReports/CrossReportEditViewSpec.jsx]: Yeah, I thought county was required. removed the assertion but not the `s`

Comment (AmanShurafa) [reply spec/javascripts/components/screenings/crossReports/CrossReportEditViewSpec.jsx]: Code is collapsed. There are 3 required fields that are tested here. Agency Name, Cross Reported on Date, and Communication Method.

**Pull request closed without merging**